spec/set-prototype-is-subset-of.html

```diff
@@ -1,19 +1,17 @@
-<p>When the `isSubsetOf` method is called with argument _iterable_, the following steps are taken:</p>
+<p>When the `isSubsetOf` method is called with argument _O_, the following steps are taken:</p>
 
 <emu-alg>
     1. Let _set_ be the *this* value.
-    1. If Type(_set_) is not Object, throw a *TypeError* exception.
-    1. If _set_ does not have a [[SetData]] internal slot, throw a *TypeError* exception.
-    1. Let _entries_ be the List that is _set_.[[SetData]].
-    1. If Type(_iterable_) is not Object, throw a *TypeError* exception.
-    1. If _iterable_ does not have [[SetData]] internal slot,
-        1. Let _otherSet_ be Construct(%Set%, _iterable_).
-    1. Else, let _otherSet_ be _iterable_.
-    1. Let _hasCheck_ be ? Get(_otherSet_, "has").
+    1. Let _iter_ be ? GetIterator(_set_).
+    1. If _iter_ is undefined, throw a *TypeError* exception.
+    1. If Type(_O_) is not Object, throw a *TypeError* exception.
+    1. Let _hasCheck_ be ? Get(_O_, "has").
```

Comment:
How much harder would it be to make it both iterate the receiver and the argument, and not use “has” at all?